lib/app/modules/uscore_v3.0.1/us_core_careplan_sequence.rb

```diff
@@ -259,8 +281,26 @@ def validate_resource_item(resource, property, value)
       test 'At least one of every must support element is provided in any CarePlan for this patient.' do
         metadata do
           id '11'
-          link 'https://build.fhir.org/ig/HL7/US-Core-R4/general-guidance.html/#must-support'
+          link 'http://www.hl7.org/fhir/us/core/general-guidance.html#must-support'
           description %(
+
+            US Core Responders SHALL be capable of populating all data elements as part of the query results as specified by the US Core Server Capability Statement.
+            This will look through all CarePlan resources returned from prior searches too see if any of them provide the following must support elements:
+
+            CarePlan.text
+
+            CarePlan.text.status
+
+            CarePlan.status
+
+            CarePlan.intent
+
+            CarePlan.category
+
+            CarePlan.category
+
```

Comment:
Why the duplication here?

Reply:
I think this is because CarePlan.category is a must support, and CarePlan.category::assessplan is also a must support. I didn't know how to handle slices before, so I just left it as careplan.category. I made another ticket to address this.